core/storage/base_model/gae_models.py

```diff
@@ -139,6 +146,11 @@ def export_data(user_id):
             'The export_data() method is missing from the '
             'derived class. It should be implemented in the derived class.')
 
+    @staticmethod
+    def get_export_method():
+        """Model is exported as a single unshared instance."""
+        return EXPORT_METHOD.SINGLE_UNSHARED_INSTANCE
```

Comment:
This is the base class, so why not raise NotImplementedError()?

Reply:
Done

Reply:
Done